lizrd/core/llm.py

```diff
@@ -258,27 +258,41 @@ def forward(self, x, **kwargs):
         return self.fn(x, **kwargs) * self.rezero_g
 
 
+class RMSNorm(nn.Module):
+    def __init__(self, dmodel, eps=1e-8):
+        super().__init__()
+        self.eps = eps
```

Comment:
this repo's convention is to not leave unattended default values in code. While one can find older code with such instances, we aim to eliminate them at some point.